ballerina/tests/10-custom-datetime-test.bal

```diff
@@ -158,3 +162,107 @@ isolated function insertIntervalWithInvalidBalType2() returns sql:Error? {
        test:assertFail("Database Error expected.");
    }
 }
+
+@test:Config {
+    groups: ["query"],
```

Comment:
Should this also belong to the `datetime` test group? I assume if it's required to run all tests related to datetime using Pgroups=datetime, this test would not run.

Also if ./gradlew test -Pgroups=query is run, an error would be generated since this test's dependency is disabled.

Reply:
added in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/13/commits/950f2228fe9fde68f2865688c03597cabbea3792